documentation/user_docs/docs/Horace_manual.rst

```diff
@@ -0,0 +1,54 @@
+Horace Manual
+#############
+
+.. toctree::
+   :caption: Horace Manual Chapters
+   :maxdepth: 1
+
+   manual/Changing_Horace_settings
+   manual/Planning_a_Horace_scan
+   manual/Generating_SQW_files
+   manual/Manipulating_and_extracting_data_from_SQW_files_and_objects
+   manual/Correcting_for_sample_misalignment
+   manual/Data_diagnostics
+   manual/Symmetrising_etc
+   manual/Unary_operations
+   manual/Binary_operations
+   manual/Plotting
+   manual/Simulation
+   manual/Multifit
+   manual/Advanced_Multifit
+   manual/Tobyfit
+
+.. toctree::
+   :caption: Horace Manual Appendix
+   :maxdepth: 1
+
+   manual/List_of_functions
+   manual/Input_file_formats
+   manual/FAQ
+
+.. toctree::
```

Comment:
Ah! Is this the feature I was looking for? I didn't realise you could have multiple `toctree`s

Reply:
Yeah... the sphinx documentation is not the most helpful... I got this from the brille docs... don't know where Greg got it from...